library/ssl_tls.c

```diff
@@ -6413,7 +6414,7 @@ static int ssl_get_next_record( mbedtls_ssl_context *ssl )
 #if defined(MBEDTLS_SSL_DTLS_CONNECTION_ID)
     ssl->in_len = ssl->in_cid + rec.cid_len;
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
-    ssl->in_iv  = ssl->in_msg = ssl->in_len + 2;
+    ssl->in_iv = ssl->in_len + 2;
```

Comment:
There is a pattern of aligning the assignments in this function. If you rework the PR, please consider keeping the extra space before the assignment operator, so that it is aligned with the previous assignment.